### PR TITLE
Tidyup, and inclusion of 2016 Q4 meeting agenda

### PR DESCRIPTION
**Pull request by propensive**
Review by @SethTisue, pending approval by the Advisory Board.

### CODE REVIEW

File: proposals/006-compile-time-serializibility-check.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# SCP-[006]: [Compile Time Check of Serializability]
+# SCP-006: [Compile Time Check of Serializability]
```
Comment (SethTisue) [inline proposals/006-compile-time-serializibility-check.md]: the square brackets around the title are extraneous too, I think

File: agendas/003-2016-q4.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+5. Proposals
+  1. Representations
+    1. SCP-007: Collaborative Redesign and Implementation of Scala 2.13's Collections Library (Adriaan Moors)
+    2. SCP-008: Maintain scala-lang, docs.scala-lang and scala.epfl.ch (Seth Tisue)
+    3. SCP-009: Improve User Experience for Builds that use only Direct Dependencies (Eugene Burmako)
+  2. Discussion
+  3. Voting
+6. Discussion Around Advisory Board Communication and Administration
+7. Community Discussion
+  1. Ensime Project Concerns (Bill Venners, Community Representative)
```
Comment (SethTisue) [inline agendas/003-2016-q4.md]: is there a new round on this, or this is an accidental leftover from a previous meeting?

the agenda that went out by email had this instead:

```
7. Community Discussion
  1. Community Feedback from Bill Venners
  2. Community Feedback from Lars Hupel
```

File: proposals/006-compile-time-serializibility-check.md (hunk 1 above)
Comment (propensive) [inline proposals/006-compile-time-serializibility-check.md]: Yep.

File: agendas/003-2016-q4.md (hunk 2 above)
Comment (propensive) [inline agendas/003-2016-q4.md]: Yes, well spotted! That was a post-copy/paste change. I'll fix the version in the repository.

File: agendas/003-2016-q4.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+1. Opening Remarks (Jon Pretty, Chairperson)
+2. Ratification of new community member
+3. Report on Scala Center Activities (Heather Miller, Executive Director)
+4. Financial Report (Heather Miller, Executive Director)
+5. Proposals
+  1. Representations
+    1. SCP-007: Collaborative Redesign and Implementation of Scala 2.13's Collections Library (Adriaan Moors)
+    2. SCP-008: Maintain scala-lang, docs.scala-lang and scala.epfl.ch (Seth Tisue)
+    3. SCP-009: Improve User Experience for Builds that use only Direct Dependencies (Eugene Burmako)
```
Comment (xeno-by) [inline agendas/003-2016-q4.md]: This should be `SCP-009: Improve User Experience for Builds that use only Direct Dependencies (Stu Hood, Eugene Burmako)`.

**Comment by SethTisue** [comment]
merging so I can link to it from the minutes. we can always make further changes

**Pull request merged**